src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -3549,6 +3551,18 @@ public long getMaxSSTableDuration()
         return metric.maxSSTableDuration.getValue();
     }
 
+    @Override
+    public boolean hasStorageAttachedIndex()
```

Comment:
Why do we need to modify a public API for such a narrow case, can't the `TotalIndexCount` metric name be used instead to check the indexes we are interested in? Am I missing something?

Reply:
Agreed. Suggested this to Sunil earlier this morning.